src/compass/s1_geocode_metadata.py

```diff
@@ -155,7 +156,7 @@ def run(cfg, burst, fetch_from_scratch=False):
 
 
 def geocode_luts(geo_burst_h5, burst, cfg, dst_group_path, item_dict,
-                 dec_factor=40):
+                 dec_factor=(20, 5)):
```

Comment:
```suggestion
                 dec_factor_x_rng=20 . dec_factor_y_az=5):
```
Let's break up for clarity.